src/fragments/lib/datastore/ios/advanced-workflows/custom-primary-key/delete-snippet.mdx

```diff
@@ -1 +1,72 @@
-## TODO (see js snippet for examples)
\ No newline at end of file
+
+A record with a custom primary key can be deleted in the following ways:
+
+With the value of the primary key:
+
+<BlockSwitcher>
+
+<Block name="Listener (iOS 11+)">
+
+```swift
+Amplify.DataStore.delete(Book.self, withIdentifier: .identifier(isbn: "12345"), where: nil) { result in
```

Comment:
let's add a commit on top of the CPK feature branch https://github.com/aws-amplify/amplify-ios/pull/1752 that will make the `where` parameter have a default value of `nil` so that the docs can be updated to remove `where: nil` at the call site

Reply:
thanks for adding this PR https://github.com/aws-amplify/amplify-ios/pull/2041